index.html

```diff
@@ -169,31 +209,52 @@
                         <p>Project Outside-IN partners will include:</p>
```

Comment:
I wonder if this entire Partners section can be made more "present tense" now that the project is underway – instead of "will create" use "is creating"? and instead of "partners will include:" say "partners include:" ? What do you think?